client/components/seo/twitter-preview/index.jsx

```diff
@@ -21,13 +21,13 @@ export class TwitterPreview extends PureComponent {
 		};
 
 		return (
-			<div className="twitter-card-preview__container">
+			<div className="twitter-preview">
 				<div className={ `twitter-card-preview twitter-card-preview__${ type }` }>
```

Comment:
The `twitter-card-preview__${ type }` styles should also be converted. The ESLint rule doesn't complain because it doesn't check the template strings, but the classnames are still wrong.

Note that the `.twitter-card-preview` class is unused and can be removed.

Reply:
Thanks, Jarda! TIL that ESLint does not complain about ones in template strings. I have updated the suggested changes in 81ae26b81340b3db18199529656d90bb92b1020d.